README

```diff
@@ -128,6 +128,10 @@ REQUIREMENTS:
 
         Required for signed Verity images support:
           CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG
+        Required to verify signed Verity images using keys enrolled in MoK:
```

Comment:
What is MoK?

Reply:
machine-owner-keyring as set up by shim

Reply:
Is it a famous abbreviation? I've never heard that. Could you mention that explicitly?